tests/TestCase/Error/ExceptionRendererTest.php

```diff
@@ -759,6 +757,7 @@ public function testMissingLayoutPathRenderSafe()
         $ExceptionRenderer->controller->response = $response;
 
         $ExceptionRenderer->render();
+        $this->assertTrue($this->called, 'Listener added was not triggered.');
```

Comment:
Here I used the naming and error message similar to those I found in `Cake\Test\TestCase\ServerTest::testBuildMiddlewareEvent()`.